_posts/en/newsletters/2023-02-22-newsletter.md

```diff
@@ -0,0 +1,212 @@
+  spender may use for nodes which did not advertise high availability.
+  Spenders maximizing payment speed (rather than low fees) would
+  preferentially choose payment paths consisting of self-identified
+  highly available nodes.
+
+    Christian Decker [replied][decker qos] with an excellent summary of
+    problems in reputation systems, including cases of self-proclaimed
+    reputation.  One of his concerns was that typical spenders won't
+    send anywhere near enough payments to frequently encounter the same
+    nodes in a large network of payment channels.  If repeat business is
+    rare anyway, then the threat of temporarily not providing repeat
+    business may not be effective.
+
+    Antoine Riard [reminded][riard boomerang] participants about a
+    alternative approach to speeding up payments---overpayment with
+    recovery.  Previously described as boomerang payments (see
+    [Newsletter #86][news86 boomerang]) and refundable ovepayments (see
+    [Newsletter #192][news192 pp]), a spender would take their
+    payment amount plus some extra money, split it into several
+    [parts][topic multipath payments], and send the parts by a variety
+    of routes.  When a sufficient number of parts has arrived to pay the
+    invoice, the receiver claims just those parts and rejects any
+    additional parts (with additional funds) that arrive later.  This
+    requires spenders wanting fast payments have some additional funds
+    in their channel but it works even some of the paths selected by the
+    spender fail.  This reduces the need for spenders to be able to
+    easily find highly available channels.  The challenge of this
+    approach is building a mechanism that prevents receivers from
+    keeping any overpayment that arrives.
+
+- **Feedback requested on LN good neighbor scoring:** Carla Kirk-Cohen
+  and Clara Shikhelman [posted][ckc-cs reputation] to the Lightning-Dev
+  mailing list to request feedback on the recommend parameters for how a
+  node judges whether its channel counterparties are a good source of
+  forwarded payments.  They suggest several criteria to judge by and
+  recommend default parameters for each criteria, but are seeking
+  feedback about the choices.
+
+    If a node determines that one of its peers is a good neighbor and
+    that neighbor tags a forwarded payment as being endorsed by it, the
+    node can give that payment access to more of its resources than it
+    gives unqualified payments.  It may also endorse that payment when
+    it forwards it to the next channel.  As described in a paper (see
+    [Newsletter #226][news226 jam]), this is part of a proposal to
+    mitigate [channel jamming attacks][topic channel jamming attacks].
+
+- **Proposed BIP for Codex32 seed encryption scheme:** Russell O'Connor
+  and Andrew Poelstra (using anagrams of their names) proposed a BIP for
+  a new scheme backing up and restoring [BIP32][] seeds.  Similar to
```

Comment:
Missing link to the mailing list post.

Reply:
https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-February/021469.html